runtime-testsuite/test/org/antlr/v4/test/runtime/go/BaseGoTest.java

```diff
@@ -165,6 +165,9 @@ private static void cacheGoRuntime(File tmpPackageDir) throws Exception {
 	}
 
 	private static void copyFile(File source, File dest) throws IOException {
+		if ( source.isDirectory() ) {
```

Comment:
In general can you point me at a style guide. The bigger style issue with this in the exist of paren around the test. To much swapping between Java and Go :-)